Integrations/integration-symantec-DLP-incident-reportintg-update.yml

```diff
@@ -0,0 +1,178 @@
+description: Integration to fetch and update symantec DLP Incidents
+detaileddescription: Symantec DLP integration uses SOAP API to communicate with Enforce
+  server to fetch and update  incident Data. uses Zeep package for SOAP Api
+configuration:
+- display: 'credentials '
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: server-ip
+  name: server-ip
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: saved report Id
+  name: saved-Report-id
+  defaultvalue: ""
+  type: 0
+  required: false
+script:
+  script: |+
+    from requests import Session
+    from zeep import Client
+    from zeep.transports import Transport
+    from requests.auth import AuthBase, HTTPBasicAuth
+    from zeep import helpers
+    import random
+    import datetime
+    import json
+    import requests
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+    requests.packages.urllib3.disable_warnings()
+
+    # hexdump:732073756468656572206b756d6172
+
+    class SymantecAuth(AuthBase):
+        def __init__(self, username, password, host):
+            self.basic = HTTPBasicAuth(username, password)
+            self.host = host
+
+        def __call__(self, r):
+            if r.url.startswith(self.host):
+                return self.basic(r)
+            else:
+                return r
+
+    serverIp=demisto.params()['server-ip']
+    username=demisto.params()['credentials']['identifier']
+    password=demisto.params()['credentials']['password']
+    savedReportId = demisto.params()['saved-Report-id']
+
+
+    wsdl = 'https://{}/ProtectManager/services/v2011/incidents?wsdl'.format(serverIp)
+
+    session = Session()
+    session.auth = SymantecAuth(username,password, "https://{}".format(serverIp))
+    session.verify = False
+    transport = Transport(session=session)
+
+    client = Client(wsdl=wsdl, transport=transport)
+
+    def  getIncidentId(dtobj):
+
+        if type(dtobj) is datetime.datetime:
+            return client.service.incidentList(savedReportId=savedReportId, incidentCreationDateLaterThan=dtobj)
+        else:
+            raise ValueError('excpected datetime object')
```

Comment:
for more nicer error message and not stack trace, use `return_error('some error here')`

Reply:
handled the exception with return_error()